migration/src/m0000250_create_sbom_package.rs

```diff
@@ -53,5 +53,6 @@ pub enum SbomPackage {
     Table,
     SbomId,
     NodeId,
+    Group,
```

Comment:
Please remove this.

Reply:
why we needs remove it?

Reply:
It it not needed in this file, the enums are per file.